spec/support/metric_helper.rb

```diff
@@ -37,9 +37,9 @@ def queue_timings(items = MiqQueue.where(:method_name => %w[perf_capture_hourly
       end
 
       # sorry, stole from the code - not really testing
-      def arg_day_range(start_time, end_time, threshold = 1.day)
-        (start_time.utc..end_time.utc).step_value(threshold).each_cons(2).collect do |s_time, e_time|
-          [s_time, e_time]
+      def arg_day_range(start_time, end_time, *args)
+        (start_time.utc..end_time.utc).step_value(1.day).each_cons(2).collect do |s_time, e_time|
+          [s_time, e_time] + args
```

Comment:
Every caller of this (at least in this PR) seems to pass `nil, false`, unless I'm missing something.  Would it make sense to just default it here if no args are passed?  It's a little confusing because I think it's directly tied to the signature for `perf_capture`, so it's not like we can pass _anything_ in there args.

Reply:
I guess what I'm saying is maybe the signature here should be something like (unless my assumptions for the signature matching is wrong):

```ruby
def arg_day_range(start_time, end_time, target_ids = nil, rollup = false) # matches the perf_capture signature
```

Reply:
thanks. looks like they are all the same thing.

Hope in the future we will not segment by dates and just merge all these into a single message.